src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({ entries, color, onLike }) => {
+  const chatComponents = entries.map((entry, i) => (
+    <ChatEntry
+      key={i}
+      id={i}
+      {...entry}
+      color={color}
+      onLike={onLike}
+    />
```

Comment:
The best practice is to only default to using an index for `key` if there is no unique id provided for the elements. In this case, our data has a unique `id` for every message, so we should use that value as both the `key` and `id` props here. Looking forward, when reading and displaying data from a database, those records typically have a unique ID field we should prefer to use.